src/Functions/toTime.cpp

```diff
@@ -10,7 +10,42 @@ using FunctionToTime = FunctionDateOrDateTimeToSomething<DataTypeDateTime, ToTim
 
 REGISTER_FUNCTION(ToTime)
 {
-    factory.registerFunction<FunctionToTime>();
+    FunctionDocumentation::Description description_toTime = R"(
+Converts a date with time to a certain fixed date, while preserving the time.
+
+If the `date` input argument contained sub-second components, they will be dropped in the returned `DateTime` value with second-accuracy.
+    )";
+    FunctionDocumentation::Syntax syntax_toTime = R"(
+toTime(date[, timezone])
+    )";
+    FunctionDocumentation::Arguments arguments_toTime = {
+        {"date", "Date to convert to a time. [`Date`](../data-types/date.md)/[`DateTime`](../data-types/datetime.md)/[`DateTime64`](../data-types/datetime64.md)."},
+        {"timezone", "Optional. Timezone for the returned value. [`String`](../data-types/string.md)."}
+    };
+    FunctionDocumentation::ReturnedValue returned_value_toTime = "Returns DateTime with date equated to `1970-01-02` while preserving the time. [`DateTime`](../data-types/datetime.md).";
```

Comment:
`Returns the time component of a date or date with time in the form of an offset to a fixed point in time (1970-01-02 at the moment).`?